src/Server/Program.fs

```diff
@@ -42,7 +65,16 @@ let main args =
             |> Option.defaultValue Database.DatabaseType.FileSystem
 
         let port = getPortsOrDefault 8085us
-        WebServer.start database clientPath port
+        
+        WebHost
+            .CreateDefaultBuilder()
```

Comment:
We have:

```
    WebHostBuilder()
        .UseKestrel()
```

Is this the same? Asking out curiosity.

Reply:
no idea.

Reply:
`CreateDefaultBuilder` is a helper that calls `UseKestrel`, `UseIISIntegration` and some other stuff.

Reply:
https://github.com/aspnet/MetaPackages/blob/dev/src/Microsoft.AspNetCore/WebHost.cs#L148

Reply:
Thanks, @vasily-kirichenko! Do you know if `UseIISIntegration` has any effect when you run the server in a no-Windows platform?